src/models/tools/geometry/jxg-point.ts

```diff
@@ -27,11 +27,24 @@ const defaultProps = {
         strokeColor: kPointDefaults.strokeColor
       };
 
+// fillColor/strokeColor are ephemeral properties that change with selection;
+// we store the desired colors in clientFillColor/clientStrokeColor for persistence.
+export function syncClientColors(props: any) {
+  const { selectedFillColor, selectedStrokeColor, ...p } = props || {} as any;
+  if (p.fillColor) p.clientFillColor = p.fillColor;
+  if (p.strokeColor) p.clientStrokeColor = p.strokeColor;
+  if (selectedFillColor) p.clientSelectedFillColor = selectedFillColor;
+  if (selectedStrokeColor) p.clientSelectedStrokeColor = selectedStrokeColor;
+  return p;
```

Comment:
This is great! Will clean up my PR, having to repeat `fillColor` in `selectedFillColor` etc. was messy.